scripts/courses/aws_1M.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
this is ok for now, but it would be nice to cut down on the amount of boilerplate, maybe by moving much of this into a `functions.sh` script and reusing it for multiple courses.